posix-mapper/build.gradle

```diff
@@ -0,0 +1,57 @@
+plugins {
+    id 'war'
+    id 'maven'
+    id 'checkstyle'
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+apply from: '../opencadc.gradle'
+
+sourceCompatibility = 11
+
+group = 'org.opencadc'
+
+war {
+    from(System.getenv('RPS') + '/resources/') {
+        include 'swagger-ui/'
+    }
+    from('.') {
+        include 'VERSION'
+    }
+
+    archiveName 'posix.war'
```

Comment:
in every other project we use the default project name for the war (eg posix-mapper). This will be slightly odd for no reason. Please use default and fix Dockerfile and README